### PR TITLE
Men 1319

### PR DESCRIPTION
**Pull request by oleorhagen**
Stores the inventory in lmdb, and only submits positive diffs to the server.

I have also refactored all stores to /store.

### CODE REVIEW

**Comment by bboozzoo** [comment]
rebase?

File: daemon_test.go [hunk 1]
```diff
@@ -113,19 +112,19 @@ func TestDaemon(t *testing.T) {
...
-	store.AssertExpectations(t)
-
-	store = &MockStore{}
-	store.On("Close").Return(errors.New("foo"))
-	assert.NotPanics(t, func() {
-		d := NewDaemon(nil, store)
-		d.Cleanup()
-	})
-	store.AssertExpectations(t)
+	// store := &store.MockStore{}
```
Comment (bboozzoo) [inline daemon_test.go]: hm??

File: mender.go [hunk 2]
```diff
@@ -472,8 +473,25 @@ func (m *mender) RunState(ctx *StateContext) (State, bool) {
 	return m.state.Handle(ctx, m)
 }
 
+// TODO - test if the ic.Close() is functioning
```
Comment (bboozzoo) [inline mender.go]: todo?

Comment (oleorhagen) [inline mender.go]: apologies :baby:

**Comment by kacf** [comment]
Closed after discussing in https://tracker.mender.io/browse/MEN-1319.

**Pull request closed without merging**